### PR TITLE
Add an open source license (MIT License)

### PR DESCRIPTION
**Pull request by simonplattner**
Without an open source license, [nobody is allowed to copy, distribute or modify the project](https://choosealicense.com/no-permission) as a whole. As the project has multiple contributors, this includes the original owner.

---

Disclaimer: Be adviced that I am a newbro to both open source and licensing. I was just browsing through some projects and it came to my attention that this project has no open source license. I decided to add one. I went with the MIT license as [chartjs/Chart.js](https://github.com/chartjs/Chart.js) uses the same.

### CODE REVIEW

**Comment by igorlukanin** [comment]
I think this is a fair point and a great idea! Thanks @simonplattner 🙌 

Cc @etimberg @kurkle @LeeLenaleee

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2014-2022 Chart.js Contributors
```
Comment (LeeLenaleee) [inline LICENSE.md:3]: [nit] should be 2017
```suggestion
Copyright (c) 2017-2022 Chart.js Contributors
```

image

**Review by LeeLenaleee — CHANGES_REQUESTED** [review]

**Comment by etimberg** [comment]
@simonbrunel are you OK with this license for the website code?

**Review by LeeLenaleee — APPROVED** [review]

**Comment by simonplattner** [comment]
I just took a look at the Contributors page. According to the number of lines changes, it might be a good idea to also ask for explicit approval from @benmccann and @kurkle.

They are both major contributors and it would probably make sense to have their explicit approval.

**Review by kurkle — APPROVED** [review]

**Review by benmccann — APPROVED** [review]

**Review by simonbrunel — APPROVED** [review]

**Pull request merged**